packages/ERTP/src/amountMath.js

```diff
@@ -115,10 +127,12 @@ const assertValueGetAssetKind = value => {
  *
  * Made available only for testing, but it is harmless for other uses.
  *
- * @param {AmountValue} value
- * @returns {MathHelpers<*>}
+ * @template {AmountValue} V
+ * @param {V} value
+ * @returns {MathHelpers<V>}
  */
 export const assertValueGetHelpers = value =>
+  // @ts-expect-error cast
```

Comment:
It's not clear why we gave `assertValueGetAssetKind` such an elaborate type if we don't benefit from it when we use it.

Reply:
Yeah it's low value at this point. I did the work as part of seeing whether I could not have to cast and I eventually timed out. But I left the types that I did figure out because they are accurate.

When adding types to existing codebases, lots of things like this shake out. You make some part more accurate and it creates problems elsewhere. The trick is to tied them up and move on.